src/pb_stub.cc

```diff
@@ -402,7 +470,40 @@ Stub::Initialize(bi::managed_external_buffer::handle_t map_handle)
           .attr("TritonPythonModel");
   deserialize_bytes_ = python_backend_utils.attr("deserialize_bytes_tensor");
   serialize_bytes_ = python_backend_utils.attr("serialize_byte_tensor");
-  model_instance_ = TritonPythonModel();
+
+  return TritonPythonModel();
+}
+
+void
+Stub::AutoCompleteModelConfig(
+    bi::managed_external_buffer::handle_t string_handle,
+    std::string* auto_complete_config)
+{
+  model_ = StubSetup();
+
+  std::unique_ptr<PbString> pb_string_shm =
+      PbString::LoadFromSharedMemory(shm_pool_, string_handle);
+
+  py::module python_backend_utils =
+      py::module_::import("triton_python_backend_utils");
+  py::object model_config =
+      python_backend_utils.attr("ModelConfig")(pb_string_shm->String());
+
+  if (py::hasattr(model_, "auto_complete_config")) {
+    model_config = model_.attr("auto_complete_config")(model_config);
+  }
+  if (py::isinstance<py::none>(model_config)) {
```

Comment:
The error checking here needs to be improved to make sure the returned object is an instance of pb_utils.ModelConfig.